pkg/assembler/graphql/schema/vulnerability.graphql

```diff
@@ -70,11 +70,18 @@ Use null to match on all values at that level.
 For example, to get all vulnerabilities in GUAC backend, use a VulnSpec
 where every field is null.
 
+Setting the noVuln boolean true will ignore the other inputs for type and vulnerabilityID.
+Setting noVuln to true means retrieving only nodes where the type of the vulnerability is "novuln" 
+and the it has an empty string for vulnerabilityID. Setting it to false means retrieving only nodes
+with identified vulnerabilities. Setting one of the other fields and omitting the noVuln means retrieving
+vulnerabilities for the corresponding type and vulnerabilityID.
```

Comment:
from what i read here false == unset, is that true?

Reply:
can i suggest
`Setting it to false filters out all results that are "novuln".`

Reply:
updated.